tox.ini

```diff
@@ -11,9 +11,10 @@ envlist =
     {py37,py38}-flake8
     {py35,py36,py37,pypy}-dj22
     {py36,py37,py38}-dj30
+    {py36,py37,py38}-dj31
     {py35,py36,py37,pypy}-djmaster
-    py38-dj30-postgres
-    py38-dj30-mysql
+    py38-dj31-postgres
```

Comment:
Please leave this on `dj30` for now... once `Django 3.1` lands I am all in favor but until then lets leave it on the latest official release.

Reply:
> once Django 3.1 lands

According to [django-developers](https://groups.google.com/g/django-developers/c/D3c9--T-i-4/m/ORVCfKrHAQAJ) this might happen tomorrow.  Since this PR doesn't change any code, I propose to wait till tomorrow and also remove the `rc1` (a few lines below).

If django's releasers decide to push another rc or if you want to merge this PR before django's release, I'll update the PR.

Reply:
Yeah (please nudge me if we get 3.1 and I forget to merge in the next couple of days :-) )

Reply:
Django 3.1 has been released and I've removed the `rc1`

Reply:
Awesome thanks !